src/snapred/backend/dao/request/ReductionRequest.py

```diff
@@ -11,7 +11,7 @@ class ReductionRequest(BaseModel):
     useLiteMode: bool
     focusGroup: Union[Optional[FocusGroup], List[FocusGroup]]
     userSelectedMaskPath: Optional[str] = None
-    version: Union[int, Literal["*"]] = "*"
+    version: Optional[int] = None
```

Comment:
Tangential, but:
1. I'm not sure version should be here yet, we dont currently have plans to support users specifying a version for reduction.  It will just refer to the index for calib/norm
2. We would probably need separate versions for norm and calib if such a feature was added

Reply:
Yeah, Kort mentioned that as well.  This value is what is used to lookup the calibration/normalization, and so definitely needs two values